protocol/avdl/keybase1/os.avdl

```diff
@@ -0,0 +1,12 @@
+@namespace("keybase.1")
+protocol runtime {
+    enum RuntimeGroup {
+        UNKNOWN_0,
+        // Linux or BSD
+        UNIXLIKE_1,
```

Comment:
Isn't darwin unix-like?  Maybe LINUXLIKE?

Reply:
I tried looked up what nix, *nix, unix-like, unix all meant and couldn't find anything that grouped linux and bsd on one side and mac on the other. I think Linuxlike is technically less accurate but maybe makes more sense for Keybase since we special case darwin, what do you think?

Reply:
I don't feel strongly about it.  You can keep it as is if you prefer it, it just caught me off-guard at first glance.